sale_credit_point/views/point_history.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <record id="view_credit_point_history_tree" model="ir.ui.view">
+        <field name="name">view.credit.point.history.tree</field>
+        <field name="model">credit.point.history</field>
+        <field name="arch" type="xml">
+            <tree>
+                <field name="partner_id" readonly="True"/>
+                <field name="operation" readonly="True"/>
+                <field name="amount" readonly="True"/>
+                <field name="credit_point_currency_id" readonly="True"/>
+                <field name="credit_point_currency_id" readonly="True" invisible="1" />
+                <field name="create_date" readonly="True"/>
+                <field name="comment" readonly="True"/>
+            </tree>
+        </field>
+    </record>
+
+    <record id="credit_point_history_action" model="ir.actions.act_window">
```

Comment:
what about filtering this by default on current year only and group  them by partner?